src/client/components/form-elements/ErrorableTextInput.jsx

```diff
@@ -66,6 +67,10 @@ class ErrorableTextInput extends React.Component {
       requiredSpan = <span className="hca-required-span">*</span>;
     }
 
+    if (this.props.charMax) {
+      maxCharacters = (<small>(Max. {this.props.charMax} characters)</small>);
```

Comment:
I was trying to prevent us from doing two things: 1. giving the user an error message that contains more than one thing in it, and 2. adding in more front-end validations when there is an html attribute that does this already.